smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CodegenVisitor.java

```diff
@@ -235,6 +235,10 @@ void execute() {
             protocolGenerator.generateProtocolTests(context);
         }
 
+        if (settings.getEnableDefaultReadme()) {
```

Comment:
CodegenVisitor is being removed as part of DirectedCodegen changes. I think we should hold this until that's merged and rebase.